pkg/exporter/ovs.go

```diff
@@ -104,6 +104,23 @@ func FilterBridge(brName interface{}) bool {
 	return false
 }
 
+func FilterNormalBridge(brName interface{}) bool {
+	datapathConfig, err := datapath.GetDatapathConfig()
+	if err != nil {
+		klog.Fatalf("Failed to get datapath config. error: %v. ", err)
+	}
+	name := brName.(string)
+	for _, ovsbrname := range datapathConfig.ManagedVDSMap {
+		policyBrName := ovsbrname + "policy"
+		clsBrName := ovsbrname + "cls"
+		uplinkBrName := ovsbrname + "uplink"
```

Comment:
use constant keyword

Reply:
has been modified

Reply:
change "policy" to datapath.POLICY_BRIDGE_KEYWORD, etc.

Reply:
constant keywords are added to the current file
<img width="447" alt="image" src="https://user-images.githubusercontent.com/75230454/161182845-21089f61-f23a-44a8-ac4c-5018b910ccb2.png">
<img width="455" alt="image" src="https://user-images.githubusercontent.com/75230454/161182925-ac7b21a5-31fc-4652-862d-195c0a379743.png">

Reply:
ok